packages/legend-query-builder/src/components/QueryBuilder_LegendApplicationPlugin.ts

```diff
@@ -41,4 +46,8 @@ export class QueryBuilder_LegendApplicationPlugin extends LegendApplicationPlugi
       QUERY_BUILDER_COMMAND_CONFIG,
     );
   }
+
+  getCheckEntitlementsEditorRender(): CheckEntitlementEditorRender | undefined {
+    return undefined;
```

Comment:
I think this is not the right place to add this function. Because this type of plugin is specific to each application `studio`,
`taxonomy`, and `query`. I think better place to add this is `LegendQueryApplicationPlugin`